knime_extension/src/nodes/spatialnetwork.py

```diff
@@ -532,6 +538,7 @@ def execute(self, exec_context: knext.ExecutionContext, input1, input2, input3):
         from pyproj import CRS  # For CRS Units check
         import logging
 
+        # Start ---------------Repeated code as those in Isochrone Map
```

Comment:
See comment below refactor code and move all of these utility methods into their own class or create private methods in this file instead of copying the code